tests/settings/test_settings.py

```diff
@@ -13,12 +13,24 @@
     'exactness'
 ]
 
+DEFAUT_TYPO_TOLERANCE =   {
+    'enabled': True,
+    'minWordSizeForTypos': {
+        'oneTypo': 5,
+        'twoTypos': 9,
+    },
+    'disableOnWords': [],
+    'disableOnAttributes': [],
+}
+
 def test_get_settings_default(empty_index):
     """Tests getting all settings by default."""
     response = empty_index().get_settings()
     assert isinstance(response, dict)
     for rule in DEFAULT_RANKING_RULES:
         assert rule in response['rankingRules']
+    for typo in DEFAUT_TYPO_TOLERANCE:
+        assert typo in response['typoTolerance']
```

Comment:
This will only check the keys and not the values, is that what you mean to do? If so it's fine.